<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 79, 176, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fixedsys</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255, 0%)</string>
   </property>
   <property name="text">
    <string>Código:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fixedsys</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255, 0%)</string>
   </property>
   <property name="text">
    <string>Fornecedor:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255, 0%)</string>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255, 0%)</string>
   </property>
   <property name="text">
    <string>Detalhes:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_codigo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>139</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 6px;
border: 1px solid #000000;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_fornecedor">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 6px;
border: 1px solid #000000;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nome">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 6px;
border: 1px solid #000000;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_detalhes">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 6px;
border: 1px solid #000000;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Fixedsys&quot;;
color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255, 0%)</string>
   </property>
   <property name="text">
    <string>Preço:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_preco">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 6px;
border: 1px solid #000000;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Arial&quot;;
color: rgb(255, 255, 255,);
background-color: rgb(255, 255, 255, 0%)</string>
   </property>
   <property name="text">
    <string>CADASTRO DE PRODUTOS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cadastrarprod">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>140</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>27</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 75 11pt &quot;Fixedsys&quot;;
background-color: rgb(244, 244, 244);
border-radius: 5px;
border: 1px solid #000000;</string>
   </property>
   <property name="text">
    <string>Cadastrar</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>230</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Fixedsys&quot;;
border-radius: 6px;
background-color: rgb(244, 244, 244);
border: 1px solid #000000;</string>
   </property>
   <property name="text">
    <string>Novo Produto</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_codigo</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>645</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_fornecedor</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_nome</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>578</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_detalhes</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>555</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_preco</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
